### PR TITLE
Gemify the style guide (from gh-pages)

### PR DESCRIPTION
**Pull request by FionaDL**
https://www.pivotaltracker.com/story/show/171624976

Copied the structure and code from
fastruby#6
fastruby#7

tested in dash and it seems to be working...
Screen Shot 2020-03-05 at 11 35 12 AM

Worked on this with @etagwerker and @cleicar

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,4 +1,41 @@
...
+
+	sprockets.context_class.class_eval do
+		def asset_path(path, options = {})
+			File.join('images', path)
+		end
+
+		def asset_url(path, options = {})
+			"url(#{asset_path(path)})"
+		end
+	end
```
Comment (etagwerker) [inline Rakefile]: @FionaDL Could you fix the spacing here?

File: javascript/main.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
+//= require jquery
+//= require jquery_ujs
+//= require popper
+
```
Comment (etagwerker) [inline javascript/main.js]: @FionaDL shouldn't this file be moved to `vendor/assets` as well? I don't think you will be able to use it from `dash` if it's here.

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
-# styleguide
+
+# Ombulabs::Styleguide
```
Comment (etagwerker) [inline README.md]: @FionaDL Please follow our style guide for this: http://ombulabs.github.io/styleguide/

It should be "OmbuLabs::StyleGuide"

Comment (FionaDL) [reply README.md]: Ah yes, I see.

File: README.md [hunk 4]
```diff
@@ -1 +1,50 @@
-# styleguide
+# Ombulabs::StyleGuide
```
Comment (etagwerker) [inline README.md]: @FionaDL It seems that you missed my comment here.

Comment (FionaDL) [reply README.md]: I did miss changing the OmbuLabs. Sorry!

**Pull request merged**